pack/eoe/eoec.json

```diff
@@ -3615,7 +3615,7 @@
         "position": 173,
         "quantity": 1,
         "stage": 3,
-        "text": "Each revealed Mist-Pylon can be attacked as if it were an enemy. Its fight value is equal to its shroud value and its health is equal to its clue value. While it has no health remaining, it is collapsed <i>(it remains in play)</>.\n[reaction] When you discover 1 or more clues at a Mist-Pylon, spend those clues: Deal that much damage to that Mist-Pylon.\n<b>Objective</b> – At the end of the round, if each Mist-Pylon is collapsed, advance this act.",
+        "text": "Each revealed Mist-Pylon can be attacked as if it were an enemy. Its fight value is equal to its shroud value and its health is equal to its clue value. While it has no health remaining, it is collapsed <i>(it remains in play)</i>.\n[reaction] When you discover 1 or more clues at a Mist-Pylon, spend those clues: Deal that much damage to that Mist-Pylon.\n<b>Objective</b> – At the end of the round, if each Mist-Pylon is collapsed, advance this act.",
```

Comment:
Please can you adjust The Nameless Madness (lines 3721 and 3722): evade 1, fight 1.

Reply:
Done!